container/build.py

```diff
@@ -16,6 +16,9 @@
 
 """Builds a toolchain container, with Google Cloud Container Builder or locally.
 
+IF THIS SCRIPT IS CALLED FROM OUTSIDE OF THE BAZEL-TOOLCHAINS REPO, THE BAZEL-TOOLCHAINS REPO
+MUST BE A SUBDIRECTORY OF THE OUTER PROJECT
```

Comment:
Add example illustrating what the correct setup would be (do you need to checkout the entire project as a subdirectory, how would you then use it?